salt/_states/metalk8s.py

```diff
@@ -10,16 +10,25 @@ def __virtual__():
     return __virtualname__
 
 
-def static_pod_managed(name, source, config_files=None, context=None):
+def static_pod_managed(name, source,
+                       config_files=None, config_files_opt=None, context=None,
```

Comment:
Hmm, you already can provide empty list, I don't really get the point

Reply:
The intent is to have empty lists in the function definition, for clarity. That way you don't need the `None` checks that are below :)

Reply:
I think I will keep the `None` because here we handle if the user give `None` as input which is the case when for example we do some jinja with a for loop in yaml file.
```
toto:
{%- for i in [] %}
  - {{ i }}
{%- endfor %}
```
"toto" will be `None`

Reply:
Indeed. Works for me then!

Reply:
Let's not do that kind of thing anyway, passing mutable defaults to functions is not recommended in Python (even if your function is supposed to be pure).